app.py

```diff
@@ -111,7 +111,7 @@ def app(
     logger.setLevel(level=map_log_level(log_level))
 
     client_s3 = boto3.client("s3")
-    client_kms = boto3.client("kms")
+    client_kms = boto3.client("kms", region_name="ap-southeast-1")
```

Comment:
Since this is an open-source repo usable by anyone, the code has to stay unbiased to any region, would prefer this to be a README instruction instead?

`boto3` is able to implicitly read off standard AWS env vars:
https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#using-environment-variables

```bash
# Remember to set your AWS profile
export AWS_PROFILE=my-aws-profile

# As well as default regin
export AWS_DEFAULT_REGION="ap-southeast-1"
```

Reply:
Ok, will edit the README in another PR